spcs-three-tier/setup.sh

```diff
@@ -1,5 +1,4 @@
 set -e
-snow sql -f "prepare/spcs_setup.sql"
```

Comment:
why are we removing this? Maybe some context would help.

Reply:
updated PR description with Jira ticket :) 
https://snowflakecomputing.atlassian.net/browse/SNOW-1527720